henryade/Banka-React
# #168078471 Implement Sign In Feature

#### What does this PR do?
Implement sign in feature

#### Description of Task to be completed?
- add components and styles
- add actions, action types, and reducers
- add custom functions

#### How should this be manually tested?
Open [Link](https://banka-app-in-react.herokuapp.com/signin) to see page.
Input your details and click login

#### Any background context you want to provide?
 - N/A
#### What are the relevant pivotal tracker stories?
[#168078471](https://www.pivotaltracker.com/story/show/168078471)
#### Screenshots (if appropriate)
![Screenshot from 2019-11-15 18-05-11](https://user-images.githubusercontent.com/47576037/68961368-87238500-07d2-11ea-83df-817511fae6e3.png)
![Screenshot from 2019-11-15 18-06-22](https://user-images.githubusercontent.com/47576037/68961427-a91d0780-07d2-11ea-8079-5e58c7d89893.png)
